util/query.py

```diff
@@ -324,24 +325,94 @@ def getNFTVolumes(
 
 
 @enforce_types
-def _filterOutPurgatory(nftvols: dict, chainID: int) -> dict:
+def _filterDids(nft_dids: List[str]) -> List[str]:
+    """
+    @description
+      Filter out DIDs that are in purgatory and are not in Aquarius
+    """
+    nft_dids = _filterOutPurgatory(nft_dids)
+    nft_dids = _filterToAquariusAssets(nft_dids)
+    return nft_dids
+
+
+@enforce_types
+def _filterOutPurgatory(nft_dids: List[str]) -> List[str]:
     """
     @description
-      Return pools that aren't in purgatory
+      Return dids that aren't in purgatory
+
+    @arguments
+      nft_dids: list of dids
+
+    @return
+      filtered_dids: list of filtered dids
+    """
+    bad_dids = _didsInPurgatory()
+    filtered_dids = set(nft_dids) - set(bad_dids)
+    return list(filtered_dids)
+
+
+@enforce_types
+def _filterNftvols(nftvols: dict, chainID: int) -> dict:
+    """
+    @description
+      Filters out nfts that are in purgatory and are not in Aquarius
 
     @arguments
       nftvols: dict of [basetoken_addr][nft_addr]:vol_amt
+      chainID: int
 
     @return
       filtered_nftvols: list of [basetoken_addr][nft_addr]:vol_amt
     """
-    bad_dids = _didsInPurgatory()
-    filtered_pools = {}
+    if chainID == networkutil.DEV_CHAINID:
+        # can't filter on dev chain:
+        return nftvols
+
+    filtered_nfts: Dict[str, Dict[str, float]] = {}
```

Comment:
This overall update looks great! Super clean.

One small thing: rename `filtered_nfts` to `filtered_nftvols`. 

Then, good to go!